scripts/exe-from-github.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# exe-from-github.sh
+#
+#   Runs an executable fetched from a github repository, on a given release & platform.
+#   This only works for executables which follow a similar naming convention, like hlint
+#   and stylish-haskell.
+#
+#   Adapted from https://raw.githubusercontent.com/ndmitchell/neil/master/misc/travis.sh
+#
+# Usage: ./exe-from-github.sh
+#
+# Available ENV vars:
+#   REPOSITORY           Name of the repository
+#   PACKAGE              Name of the release package
+#   PLATFORM             Target platform of the executable
+#   VERSION              Executable version number
+#   RELEASE              Release version
```

Comment:
I see there's some difference between these, but I don't understand exactly what. Is release a tag name? Could this be clarified a bit?

I guess `$PACKAGE-$VERSION-$PLATFORM.tar.gz` is the name of the asset e.g. from here: https://github.com/ndmitchell/hlint/releases

Reply:
Perhaps just this helps:
```suggestion
#   VERSION              Executable version number
#   RELEASE              Release git tag
```
?